app/classifier/tasks/survey/chooser.cjsx

```diff
@@ -2,6 +2,11 @@ React = require 'react'
 TriggeredModalForm = require 'modal-form/triggered'
 sortIntoColumns = require 'sort-into-columns'
 
+# key codes
+BACKSPACE = 8
```

Comment:
Turns out [Google removed backspace in Chrome 52](http://venturebeat.com/2016/08/14/restore-backspace-shortcut-chrome/), which is why I missed this.